BrainPortal/app/views/nh_messages/new.html.erb

```diff
@@ -0,0 +1,62 @@
+
+<%-
+#
+# Neurohub Project
+#
+# Copyright (C) 2019-2020
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+<% title 'New Message', "" %>
+<div class="nh_content">
+  <div class="nh_form">
+    <%= error_messages_for @message, :header_message => "Message could not be send." %>
+    <p class="nh_form_title">Send Message</p>
+
+    <%= form_for @message, :as => :message, :url => { :action => "create" }, :datatype => "script" do |f| -%>
+
+      <fieldset>
+      To all members of project ( or an individual user ): <%= group_select(:group_id,
+           {
+             :groups  => @recepients,
+             :selector => @group_id, # :group_id is not an attribute of Message.
+           },
+           { :include_blank => true }
+           )
+      %>
+      </fieldset>
+      <fieldset>
+        <%= f.label :header %>
+        <%= f.text_field :header, :size => 92 %>
+      </fieldset>
+      <fieldset>
+        <%= f.label :description, 'Brief Topic Description' %>
+        <%= f.text_area :description, :rows => 5, :cols => 80 %>
+      </fieldset>
+      <fieldset>
+        <%= f.label :variable_text, 'Message' %>
+        <%= f.text_area :variable_text, :rows => 5, :cols => 80 %>
+      </fieldset>
+      <div class="btn-section">
+        <%= f.submit "Send", :class=>"btn-solid primary" %>
+      </div>
+    </p>
+  </div>
+<% end %>
```

Comment:
Indentation is all messed up; missing newline on last line

Reply:
I tried, but might need hint what to change. I cannot guess what you template indentation expectation.